src/Routing.tsx

```diff
@@ -210,6 +212,10 @@ const getRoutes = ({ enableServiceAccounts, isITLess }: Record<string, boolean>)
       },
     ],
   },
+  {
+    path: '/access-management/users-and-user-groups/*',
+    element: UsersAndUserGroups,
+  },
```

Comment:
```suggestion
  {
    path: '/users-and-user-groups/*',
    element: UsersAndUserGroups,
  },
```

this should help - you don't have to include the `access-management` section.

Re: "Before it was going to that page and not displaying my component there, but now it's automatically redirecting me from https://stage.foo.redhat.com:1337/iam/access-management/users-and-user-groups to https://stage.foo.redhat.com:1337/iam/user-access/users?status=Active&per_page=20&page=1 when the 'Users & User Groups' link is clicked."

This was because of a misconfiguration on the navigation side - I've opened a [PR](https://github.com/RedHatInsights/chrome-service-backend/pull/657) yesterday fixing that. Hope that helps